UM/Application.py

```diff
@@ -101,6 +101,12 @@ def __init__(self, name: str, version: str, build_type: str = "", **kwargs):
 
         self._operation_stack = OperationStack(self.getController())
 
+        # Ensure configuration folder has a folder for themes
+        try:
+            os.makedirs(os.path.join(Resources.getStoragePath(Resources.Resources), "themes"))
```

Comment:
Seeing as how themes are listed dynamically now instead of being hard-coded, it feels to me like they are ordinary resources and should be put in the `Resources` class.